### ISSUE TITLE
conda package

### ISSUE THREAD

**Issue by matuskosut** [statgen/savvy#10]
Hi @jonathonl 

I created a conda package of savvy https://anaconda.org/bioconda/savvy

are you interested in adding some documentation about it in readme?
For example mention installation:
```
conda install -c bioconda savvy
```

and maybe add some fancy badge? [![Savvy](https://anaconda.org/bioconda/savvy/badges/installer/conda.svg)](https://anaconda.org/bioconda/savvy)
```
[![Savvy](https://anaconda.org/bioconda/savvy/badges/installer/conda.svg)](https://anaconda.org/bioconda/savvy)
```

**Comment by jonathonl** [comment]
Hey, sorry for the delayed response. 

Given that Savvy is on a new major version now, I'd like to update the conda recipe to reflect the new version prior to including the badge.

In regard to the edits at  https://github.com/bioconda/bioconda-recipes/blob/master/recipes/savvy/patches/0001-patch-cmakelists-for-conda-build.patch, I'm surprised that any of them are necessary. Were there specific issues you were encountering that spurred these changes? I've never contributed to conda before, so I'm unaware of the potential roadblocks.

Setting `-DCMAKE_PREFIX_PATH:PATH=${PREFIX}` and `-DCMAKE_INSTALL_PREFIX:PATH=${PREFIX}` should be enough for CMake to find the correct include path / libraries and install to the correct locations. Also, is there a conda policy against statically linking executables?

**Comment by matuskosut** [comment]
Hi @jonathonl 

sorry I was a bit busy myself too. Feel free to submit a PR, if you ping me I can help with a review.

Static linking may work, but is not recommended. Don't remember what were the issues that lead me to that - there might be a simpler way for sure - good to test their CI pipelines. It is easy to fetch prebuilt bioconda package from pull request and give it a test run.

**Comment by jonathonl** [comment]
Sounds good. Thanks for the info.

**Comment by jonathonl** [comment]
Thanks @matuskosut.

Resolved with 52aa24915d9569f914eec3d102b60e21e394530b
